src/main/java/com/iota/iri/TransactionValidator.java

```diff
@@ -10,27 +10,24 @@
 import com.iota.iri.model.TransactionHash;
 import com.iota.iri.network.TransactionRequester;
 import com.iota.iri.storage.Tangle;
-import com.iota.iri.zmq.MessageQ;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.util.*;
 import java.util.concurrent.atomic.AtomicBoolean;
-import java.util.concurrent.atomic.AtomicInteger;
 
 import static com.iota.iri.controllers.TransactionViewModel.*;
```

Comment:
I'll merge now (because this simple PR was waiting around long enough).
However that I will soon I will add `AvoidStaticImports` rule to checkstyle.
I will start to put in an effort to define the style rules of this project.